### PR TITLE
HEEDLS-895 : UAR - Fix for broken migration

### PR DESCRIPTION
**Pull request by GeorgeBlackwell**
### JIRA link
_[HEEDLS-895](https://softwiretech.atlassian.net/browse/HEEDLS-895)_

### Description
_Missed a couple of things after incorrectly running the migration locally. This should hopefully fix it._

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [X] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [ ] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [X] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs [hunk 1]
```diff
@@ -37,6 +39,14 @@ public override void Down()
             Rename.Column("Email_deprecated").OnTable("AdminAccounts").To("Email");
             Rename.Column("Email_deprecated").OnTable("DelegateAccounts").To("Email");
 
+            Create.Index("IX_AdminAccounts_CentreId_Email").OnTable("AdminAccounts").OnColumn("CentreID")
+                .Ascending()
+                .OnColumn("Email").Ascending().WithOptions().NonClustered();
+
+            Create.Index("IX_DelegateAccounts_CentreId_Email").OnTable("DelegateAccounts").OnColumn("CentreID")
+                .Ascending()
+                .OnColumn("Email").Ascending().WithOptions().NonClustered();
```
Comment (AlexJacksonDS) [inline DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs:48]: This isn't going to create the correct if not null indexes, you need the ones I created in the previous Up

File: DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs [hunk 2]
```diff
@@ -9,21 +9,23 @@ public override void Up()
         {
             Create.Table("UserCentreDetails")
                 .WithColumn("ID").AsInt32().NotNullable().PrimaryKey().Identity()
-                .WithColumn("UserID").AsInt32().NotNullable().ForeignKey("Users", "ID")
-                .WithColumn("CentreID").AsInt32().NotNullable().ForeignKey("Centres", "ID")
+                .WithColumn("UserID").AsInt32().NotNullable().ForeignKey("Users", "ID").WithColumn("CentreID").AsInt32().NotNullable().ForeignKey("Centres", "CentreId")
```
Comment (AlexJacksonDS) [inline DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs:12]: What happened to the formatting here?

**Review by AlexJacksonDS — COMMENTED** [review]
Couple of things.

**Review by SteveJacksonSoft — COMMENTED** [review]
Nothing to add!

**Review by SteveJacksonSoft — APPROVED** [review]
Yup, looks good!

**Pull request merged**